src/utils/components/PolicyForm/PolicyWizard/InterfacesStep.tsx

```diff
@@ -40,61 +36,55 @@ const InterfacesStep: FC<InterfacesStepProps> = ({ policy, setPolicy, error }) =
         };
       }
 
-      draftPolicy.spec.desiredState.interfaces.unshift({
-        type: InterfaceType.LINUX_BRIDGE,
-        name: `interface-${draftPolicy.spec.desiredState.interfaces.length}`,
+      const newInterface: NodeNetworkConfigurationInterface = {
+        type: interfaceTypes?.[0],
+        name: `${interfaceTypes?.[0]}-${draftPolicy.spec.desiredState.interfaces.length}`,
         state: 'up',
-        bridge: {
+      };
+
+      if (bindingStep)
+        newInterface.bridge = {
           options: {
             stp: {
               enabled: false,
             },
           },
-        },
-      } as NodeNetworkConfigurationInterface);
+        };
+
+      if (interfaceTypes?.[0] === InterfaceType.BOND) {
+        newInterface['link-aggregation'] = {
+          mode: NodeNetworkConfigurationInterfaceBondMode.BALANCE_RR,
+          port: [],
+        };
+      }
+
+      draftPolicy.spec.desiredState.interfaces.push(newInterface);
```

Comment:
Consider adding a selector for this. Also, do you know if this path is guaranteed to exist? We may need optional chaining operators here.